recipes/pyorgmode/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.1" %}
+{% set name = "PyOrgMode" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cd704c620a5873bb0f943a86e62b4c6e831d4a569665f886b05327da6f84ef34
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+
+test:
+  imports:
+    - {{ name }}
+
+about:
+  home: https://github.com/bjonnh/PyOrgMode
```

Comment:
@gonvaled and @bjonnh , could you please help us sort out which is the primary source for `PyOrgMode`?